test/unittests/execute_floating_point_test.cpp

```diff
@@ -100,13 +88,140 @@ struct WasmTypeName
     }
 };
 
+template <typename T>
+class execute_floating_point_types : public testing::Test
+{
+protected:
+    using L = typename FP<T>::Limits;
```

Comment:
I would prefer this to be `Limits`

Reply:
It cannot be because it will conflict of other `using Limits` in the tests.

Reply:
Why can't it use this one in the tests?

Reply:
Because each `TYPED_TEST` inherits a template `execute_floating_point_types<TypeParam>` you have to access it as `typename execute_floating_point_types<TypeParam>::Limits`. Not very short.